src/app/views/collections/create/CollectionCreateController.jsx

```diff
@@ -448,7 +448,7 @@ export class CollectionCreateController extends Component {
 
     handle409SubmitStatus(error) {
         if (error.body.message.includes("A collection with this name already exists")) {
-            log.add(eventTypes.runtimeWarning, {message: "409 response because there was an attempt to create a collection with an existing collection name: " + this.state.newCollectionDetails.name.value});
+            log.event(`Error creating collection: collection name (${this.state.newCollectionDetails.name.value}) already exists`, log.error(error))
```

Comment:
we should use the `data` attribute for the collection name (ID?) rather than concatenating it into the event name

Reply:
We don't have an ID for the collection at this point. It's not created yet so doesn't have one, and Zebedee doesn't return the ID of the collection it clashes with